<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadAudioFileWgtClass</class>
 <widget class="QWidget" name="LoadAudioFileWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1538</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoadAudioFileWgt</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="title">
        <string>waveCht</string>
       </property>
       <layout class="QVBoxLayout" name="VLayout_loadWavCht"/>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_progressState">
        <property name="title">
         <string>进程状态</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_ffmpegState">
           <property name="text">
            <string>ffmpeg状态：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_ffprobeState">
           <property name="text">
            <string>ffprobe状态：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_selLoadFile">
        <property name="text">
         <string>选择导入文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_loadState">
        <property name="text">
         <string>导入状态：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_selFileTag">
        <property name="title">
         <string>文件标签</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_selFileName">
           <property name="text">
            <string>文件名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_selFilePath">
           <property name="text">
            <string>文件路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_selFileType">
           <property name="text">
            <string>文件类型：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_selFileChannelNum">
           <property name="text">
            <string>通道数：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_selFileDuration">
           <property name="text">
            <string>持续时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_selFileBitRate">
           <property name="text">
            <string>比特率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_selFileBitrate">
           <property name="text">
            <string>采样率：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_selFileBitPerSample">
           <property name="text">
            <string>比特深度：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_loadData">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_loadWavFile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>导入wav文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_transferIntoWavAndLoad">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>转换成wav格式并导入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_autoGenSTFTResult">
        <property name="text">
         <string>自动衔接STFT分析</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxDeviceState">
        <property name="title">
         <string>设备状态</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_currentInputDevice">
           <property name="text">
            <string>输入设备：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_currentOutputDevice">
           <property name="text">
            <string>输出设备：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_showAudioDeviceDialog">
        <property name="text">
         <string>显示设备管理器</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
